src/nativeAssetManager.js

```diff
@@ -333,16 +334,18 @@ export function newNativeAssetManager(win, pubUrl) {
   /**
    * Replaces occurrences of native placeholder values with their actual values
    * in the given document.
+   * If there's no actual value, the placeholder gets replaced by an empty string.
    */
   function replace(document, { assets, adId }) {
     let html = document;
 
-    (assets || []).forEach(asset => {
-      const flag = (typeof win.pbNativeData !== 'undefined');
-      const searchString = (adId && !flag) ? `${NATIVE_KEYS[asset.key]}:${adId}` : ((flag) ? '##'+`${NATIVE_KEYS[asset.key]}`+'##' : `${NATIVE_KEYS[asset.key]}`);
+    scanForPlaceholders().forEach(placeholder => {
+      const flag = true;
```

Comment:
yes, except why is this fixed to true now?

Reply:
Oh, that's a mistake, sorry! I fixed it.